### ISSUE TITLE
Henry stickmin mod fnf! 

### ISSUE THREAD

**Issue by NeonNotFound** [luckydog7/Funkin-android#286]
![Screenshot_2021-04-10-23-41-39-49_d041af62a2d22fd7fcc1e7e386524cd6](https://user-images.githubusercontent.com/81550372/114275854-b9342780-9a56-11eb-9cb1-ead70fc2acd4.jpg)

I will send the mod link later i just finished testing it!

**Comment by NeonNotFound** [comment]
The person who added the mod on android is here! #280

**Comment by NeonNotFound** [comment]
It has the hitbox update and its working on low end devices

**Comment by MrTks216** [comment]
Nice, Thank for making lite version of this mod.

**Comment by NeonNotFound** [comment]
> Nice, Thank for making lite version of this mod.

Np i just wanted to help for the people who has low end devices

**Comment by NeonNotFound** [comment]
> ![Screenshot_2021-04-10-23-41-39-49_d041af62a2d22fd7fcc1e7e386524cd6](https://user-images.githubusercontent.com/81550372/114275854-b9342780-9a56-11eb-9cb1-ead70fc2acd4.jpg)
> 
> I will send the mod link later i just finished testing it!

THE LITE VERSION IS HERE DOWNLOAD IT THE LINK IS READY!!
https://www.mediafire.com/file/iffg4lej5nrdqs4/Henry+stickmin+mod+fnf+low+end+phones+version.apk/file

**Comment by MrTks216** [comment]
> > ![Screenshot_2021-04-10-23-41-39-49_d041af62a2d22fd7fcc1e7e386524cd6](https://user-images.githubusercontent.com/81550372/114275854-b9342780-9a56-11eb-9cb1-ead70fc2acd4.jpg)
> > 
> > I will send the mod link later i just finished testing it!
> 
> THE LITE VERSION IS HERE DOWNLOAD IT THE LINK IS READY!!
> https://www.mediafire.com/file/iffg4lej5nrdqs4/Henry+stickmin+mod+fnf+low+end+phones+version.apk/file

Hey there is problem on week 1 third song

**Comment by MrTks216** [comment]
I check it out, the third song is for week 2

**Comment by NeonNotFound** [comment]
> > > ![Screenshot_2021-04-10-23-41-39-49_d041af62a2d22fd7fcc1e7e386524cd6](https://user-images.githubusercontent.com/81550372/114275854-b9342780-9a56-11eb-9cb1-ead70fc2acd4.jpg)
> > > I will send the mod link later i just finished testing it!
> > 
> > 
> > THE LITE VERSION IS HERE DOWNLOAD IT THE LINK IS READY!!
> > https://www.mediafire.com/file/iffg4lej5nrdqs4/Henry+stickmin+mod+fnf+low+end+phones+version.apk/file
> 
> Hey there is problem on week 1 third song

Whats the problem its working on my phone the thrid song is also fine

**Comment by MrTks216** [comment]
Let me fix it

**Comment by MrTks216** [comment]
> I check it out, the third song is for week 2

My version fnf mod third song just crashed

**Comment by MrTks216** [comment]
Nevermind it worked again

**Comment by MrTks216** [comment]
I hate it sometimes do like this...Ugh

**Comment by MrTks216** [comment]
The third song are you play are easy, normal or hard may i ask?

**Comment by NeonNotFound** [comment]
> The third song are you play are easy, normal or hard may i ask?

Im playing it on normal and making it slow in debug mode

**Comment by MrTks216** [comment]
I find out the coding needs to fix, i fix now, it work well

**Comment by MrTks216** [comment]
Do you need file that i fix it the third song? I just make sure it work well

**Comment by NeonNotFound** [comment]
> Do you need file that i fix it the third song? I just make sure it work well

Yes i guess i tried to fix the problem the third song in week 1 is crashing

**Comment by MrTks216** [comment]
Here:
https://mega.nz/folder/VtAWTJRa#Ue0X2G1EHkcsfDGX_Ql2SQ

**Comment by MrTks216** [comment]
Just replace it, it will work

**Comment by MrTks216** [comment]
I need update my version fnf of this mod

**Comment by MrTks216** [comment]
Does it work now?

**Comment by NeonNotFound** [comment]
> Does it work now?

Oh sorry im charging and its night here because im from Philippines

**Comment by MrTks216** [comment]
> > Does it work now?
> 
> Oh sorry im charging and its night here because im from Philippines

Oh i see, i done updated my version of this mod

**Comment by NeonNotFound** [comment]
> > > Does it work now?
> > 
> > 
> > Oh sorry im charging and its night here because im from Philippines
> 
> Oh i see, i done update my version of this mod

Ok cya its 1:19 am here

**Comment by MrTks216** [comment]
> > > > Does it work now?
> > > 
> > > 
> > > Oh sorry im charging and its night here because im from Philippines
> > 
> > Oh i see, i done update my version of this mod
> 
> Ok cya its 1:19 am here

Ya same here, cya
